docs/en/api/renderToString.md

```diff
@@ -25,10 +25,12 @@ Renders a component to HTML.
 
 `renderToString` uses [`vue-server-renderer`](https://ssr.vuejs.org/en/basic.html) under the hood, to render a component to HTML.
 
+`renderToString` is exported from the `@vue/server-test-utils` package.
```

Comment:
IMHO, Since the code is `import { renderToString } from '@vue/server-test-utils'`, I think this is better.
You import `renderToString` from the `@vue/server-test-utils` package.

Reply:
That sounds strange to me. How about:

> `render` is part of the `@vue/server-test-utils` package.

or

>`render` is included in the `@vue/server-test-utils` package.